### PR TITLE
DOC-6570: JDK version update

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

File: content/en/platform/corda/1.6/cenm/cenm-support-matrix.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ CENM currently supports the following databases:
 
 ## JDK support
 
-Corda Enterprise Edition 4.11+ and Corda Enterprise Network Manager 1.6+ have been tested and verified to work with **Oracle JDK 8 JVM 8u381** and **Azul Zulu Enterprise 8u382**. For the Azure deployment downloadable, go to [Azul Systems](https://www.azul.com/downloads/azure-only/zulu/).
+Corda Enterprise Edition 4.12 and Corda Enterprise Network Manager 1.6+ have been tested and verified to work with **Oracle JDK 8 JVM 8u422** and **Azul Zulu Enterprise 8u422**. For the Azure deployment downloadable, go to [Azul Systems](https://www.azul.com/downloads/azure-only/zulu/).
```
Comment (tomstark99) [inline content/en/platform/corda/1.6/cenm/cenm-support-matrix.md:57]: Just change the first sentence to: 
> Corda Enterprise Network Manager 1.6+ has been tested and verified to work with **Azul Zulu Enterprise 8u422**.

Comment (AnnaK-R3) [reply content/en/platform/corda/1.6/cenm/cenm-support-matrix.md]: updated

File: content/en/platform/corda/1.6/cenm/cenm-support-matrix.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ CENM currently supports the following databases:
 
 ## JDK support
 
-Corda Enterprise Edition 4.11+ and Corda Enterprise Network Manager 1.6+ have been tested and verified to work with **Oracle JDK 8 JVM 8u381** and **Azul Zulu Enterprise 8u382**. For the Azure deployment downloadable, go to [Azul Systems](https://www.azul.com/downloads/azure-only/zulu/).
+Corda Enterprise Network Manager 1.6+ have been tested and verified to work with **Oracle JDK 8 JVM 8u422** and **Azul Zulu Enterprise 8u422**. For the Azure deployment downloadable, go to [Azul Systems](https://www.azul.com/downloads/azure-only/zulu/).
```
Comment (tomstark99) [inline content/en/platform/corda/1.6/cenm/cenm-support-matrix.md:57]: Oracle 8u421 not 8u422

Comment (AnnaK-R3) [reply content/en/platform/corda/1.6/cenm/cenm-support-matrix.md]: updated

**Review by tomstark99 — APPROVED** [review]
LGTM (one final time :)

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**